fiftyone/core/map/mapper.py

```diff
@@ -25,38 +25,41 @@
 
 import fiftyone.core.config as focc
 import fiftyone.core.map.batcher as fomb
-import fiftyone.core.sample as fos
+
+# import fiftyone.core.sample as fos
 import fiftyone.core.utils as fou
 from fiftyone.core.map.typing import SampleCollection
 
 T = TypeVar("T")  # Sample type
 R = TypeVar("R")  # Return value type of map_fcn
-U = TypeVar("U")  # Return value type of iter_fcn if set - must be same as input type to map_fcn.
+U = TypeVar(
+    "U"
+)  # Return value type of iter_fcn if set - must be same as input type to map_fcn.
 
 
 logger = logging.getLogger(__name__)
 
 
-def check_if_return_is_sample(
-    sample_collection: SampleCollection[T], map_fcn: Callable[[T], R]
-) -> bool:
-    """
-    Check if the map function returns a sample
-    """
+# def check_if_return_is_sample(
```

Comment:
@j053y idk if we just want to delete this or leave it? If we do leave it, prob should have a comment

Reply:
The comment is where it is used below. I just commented out the method itself as it was causing circular dependency errors with the imports and types it uses.